cpp/bench/ann/src/raft/raft_benchmark.cu

```diff
@@ -236,6 +251,8 @@ void parse_search_param(const nlohmann::json& conf,
       THROW("Invalid value for algo: %s", tmp.c_str());
     }
   }
+  if (conf.contains("graph_mem")) { param.graph_mem = parse_allocator(conf.at("graph_mem")); }
```

Comment:
Can we spell out `*_memory_type` to match the other properties that we parse for other algorithms? Will get confusing to set these if we use different spellings.